src/main/java/com/zzpj/backend/controllers/AlcoholController.java

```diff
@@ -0,0 +1,51 @@
+package com.zzpj.backend.controllers;
+
+import com.zzpj.backend.entities.Alcohol;
+import com.zzpj.backend.services.AlcoholServiceLocal;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+
+@RestController
+@RequestMapping("/alcohols")
+public class AlcoholController {
```

Comment:
First of all, change your branch name to something more meaningful: `add/in_memory_db_and_alcohol_crud `

Reply:
Also inti as a pull request name is misleading as well